lib/Hakyll/Web/Pandoc.hs

```diff
@@ -84,7 +86,10 @@ writePandoc = writePandocWith defaultHakyllWriterOptions
 writePandocWith :: WriterOptions  -- ^ Writer options for pandoc
                 -> Item Pandoc    -- ^ Document to write
                 -> Item String    -- ^ Resulting HTML
-writePandocWith wopt = fmap $ writeHtmlString wopt
+writePandocWith wopt (Item itemi doc) =
+    case runPure $ writeHtml5String wopt doc of
+        Left (PandocSomeError err)  -> Item itemi $ "Hakyll.Web.Pandoc.writePandocWith: unknown error: " ++ err
```

Comment:
Maybe use `fail $ "Hakyll.Web.Pandoc.writePandocWith: unknown error: " ++ err`?

Reply:
I tried this, but `Item` is not a `Monad` and doesn't have an implementation of `fail`.